src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -762,7 +762,7 @@ spanning_coe_le_of_le (delete_edges_le s)
 
 end delete_edges
 
-/-! ### Induced subgraphs -/
+/-! #### Induced subgraphs -/
```

Comment:
```suggestion
/-! ### Induced subgraphs -/
```
It looks like perhaps git did something weird when merging? (Usually sectioning in a module is at level 3, but maybe you have some other intent?)